docs/data-sources/inventory.md

```diff
@@ -45,14 +45,16 @@ output "inventory_details" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
+### Optional
```

Comment:
Was this auto generated? 

At least one of these 3 fields is required right?

Reply:
Technically that might be the only option - making them all optional to support lookups. But I think we should update the docs (possibly in the template that updates this file) to make it clear that you either need id or name+org

Reply:
Yes, agree, the docs should be updated to reflect this.

Reply:
I see this conversation was resolved but there weren't any changes. Was there a resolution?